tests/src/Mod/Part/App/TopoShapeMakeShapeWithElementMap.cpp

```diff
@@ -340,3 +340,45 @@ TEST_F(TopoShapeMakeShapeWithElementMapTests, findMakerOpInElementMap)
         }
     }
 }
+
+std::string composeTagInfo(const MappedElement& element, const TopoShape& shape)
+{
+    std::string elementNameStr {element.name.constPostfix()};
+    std::string tagInfo = POSTFIX_TAG + std::to_string(shape.Tag);
+    tagInfo +=
+        ":" + std::to_string(elementNameStr.substr(0, elementNameStr.find(tagInfo)).length());
+
+    return tagInfo;
+}
+
+TEST_F(TopoShapeMakeShapeWithElementMapTests, findTagInfoInMappedName)
+{
+    // Arrange
+    auto [cube1, cube2] = PartTestHelpers::CreateTwoCubes();
+    std::vector<Part::TopoShape> sources {cube1, cube2};
+    sources[0].Tag = 1;  // setting Tag explicitly otherwise it is likely that this test will be
+                         // more or less the same of nonMappableSources
+    sources[1].Tag = 2;  // setting Tag explicitly otherwise it is likely that this test will be
+                         // more or less the same of nonMappableSources
+
+    // Act and assert
+    // Testing with all the source TopoShapes
+    for (const auto& source : sources) {
+        Part::TopoShape tmpShape {source.getShape()};
+        tmpShape.makeShapeWithElementMap(source.getShape(), *Mapper(), sources);
+
+        // For all the mappedElements ...
+        for (const auto& mappedElement : tmpShape.getElementMap()) {
```

Comment:
I think this test looks good.  Note that if getElementMap() returns empty, then no testing will be done at all.  getElementMap() is tested separately, so not necessarily required here.  I think I'd add an expect for the length of the element map just as a compulsive double check that the test is doing something.

Reply:
Thanks for the hint!

I'll add a check right now.